PIL/Image.py

```diff
@@ -1049,6 +1108,7 @@ def crop(self, box=None):
         return self._new(self._crop(self.im, box))
 
     def _crop(self, im, box):
+        # type: (Image, LURD) -> Image
```

Comment:
This is actually a core Imaging object, which is not an Image.Image but rather a PIL._imaging object.